sys/aarch64/timer.c

```diff
@@ -79,13 +79,14 @@ static int arm_timer_attach(device_t *dev) {
     .tm_max_period = bintime_mul(HZ2BT(freq), 1LL << 30),
   };
 
-  state->intr_handler =
-    INTR_HANDLER_INIT(arm_timer_intr, NULL, dev, "ARM CPU timer", 0);
+  state->irq_res =
```

Comment:
`assert(state->intr_handler)`

Reply:
Let it crash happily - no need for assert.